website/content/glossary/_index.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Environmental Sustainability Glossary
+linkTitle: Glossary
+list_pages: true
+menu:
+  main:
+    weight: 30
+description: This glossary aims to define the wording used within our Technical Advisory Group (TAG) for environmental sustainability. It doesn't claim to be holistically complete and therefore provides additional resources where needed.
+---
+
+> This glossary is also a living artifact that changes and is edited over time.
+
+## C
+
+#### **Carbon or Carbon Dioxide (CO2)**
+
+Carbon Dioxide commonly just named carbon is a colorless, odorless gas, that is released from burning fossil fuels, such as coal, oil, and natural gas. 
+CO2 emissions are fundamentally a natural process, but one that has been exacerbated by the burning of natural resources for energy production or logistics to such an extent that it is now a major factor in climate change. Depending on how the supply chain of hardware (e.g. computer chips, cables etc.) and software (e.g. digital services like cloud computing, file sharing etc.) production/services is structured, CO2 gets emitted. 
+
+#### **Carbon Diet**
+
+Actions to reduce the output of CO2 to reduce the impact on the environment.
+
+
+#### **Carbon Footprint**
+
+Is the total amount of greenhouse gasses, often carbon dioxide, that a person, family, company, organization, or building (like a data center) releases to the environment. 
+This includes the release of GHG through direct use (e.g. the energy consumption of a data center) and indirect use (e.g. the energy required to produce servers).
+
+#### **Climate or Carbon Neutrality**
+
+Often used by organizations and companies to express that they equalize the cause of carbon pollution they generate. 
+This typically happens by buying negative carbon emissions from organizations that really saved carbon or are net-zero or even reduce more carbon than they produce. 
+However, it is important to understand that this is only a compensation and does not have to imply a new product development leading to more sustainability. Please also read about the term "Net Zero" as both terms are often mixed or misleadingly used.
+
+## E
+
+#### **Eco-Efficiency**
+
+Goods and services that use fewer resources and create less waste and pollution.
+
+#### **Emissions**
+
+An umbrella term under which for example also greenhouse gasses are covered. But emissions include other factors like micro particles that cause pollution. Emissions are categorized into three scopes:
+Scope 1: direct emissions: the CO2 output of the vehicles owned by the company
```

Comment:
Request to add "for example" when giving examples for these three, so people do not think it limited just to the direct item.  so for this one it would be:

Scope 1: direct emissions: for example the CO2 output of the vehicles owned by the company